Makefile

```diff
@@ -200,6 +224,8 @@ release: tag_major_minor
 	docker push $(NAME)/node-base:$(MAJOR_MINOR_PATCH)
 	docker push $(NAME)/node-chrome:$(MAJOR_MINOR_PATCH)
 	docker push $(NAME)/node-firefox:$(MAJOR_MINOR_PATCH)
+	docker push $(NAME)/node-opera:$(MAJOR_MINOR_PATCH)
+	docker push $(NAME)/node-phantomjs:$(MAJOR_MINOR_PATCH)
```

Comment:
Why is `node-phantomjs` here? This image was deprecated a while ago.

Reply:
Deleted.